libs/librepcb/workspace/settings/workspacesettingsdialog.ui

```diff
@@ -280,6 +280,143 @@
        </item>
       </layout>
      </widget>
+     <widget class="QWidget" name="externalApplicationsTab">
+      <attribute name="title">
+       <string>External Applications</string>
+      </attribute>
+      <layout class="QVBoxLayout" name="verticalLayout_2">
+       <item>
+        <widget class="QGroupBox" name="pdfBox">
+         <property name="title">
+          <string>PDF Reader</string>
+         </property>
+         <layout class="QGridLayout" name="gridLayout">
```

Comment:
For this use-case, a `QFormLayout` is actually more suitable than `QGridLayout`. And the horizontal spacer should be removed :wink: As an example, you could take a look at the other tabs of that dialog.

Reply:
Done